main.py

```diff
@@ -1,10 +1,60 @@
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
+import json
```

Comment:
Please sort imports: 1) build-in imports, 2) third-party imports, 3) your own imports

Reply:
@Vitalii2906 not fixed